engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/EngineJobExecutionActor.scala

```diff
@@ -647,35 +669,29 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     s"$workflowIdForLogging-BackendCacheHitCopyingActor-$jobTag-${cacheResultId.id}"
   }
 
-  private def publishHitFailure(cache: EJEACacheHit, failure: Throwable) = {
-    import WomValueSimpleton._
-    import cromwell.services.metadata.MetadataService._
-
-    cache.details foreach { details =>
-      val metadataKey = MetadataKey(
-        workflowIdForLogging,
-        Option(MetadataJobKey(jobDescriptorKey.call.fullyQualifiedName, jobDescriptorKey.index, jobDescriptorKey.attempt)),
-        s"$callCachingHitFailures[${cache.hitNumber}]:${details.escapeMeta}"
-      )
-
-      serviceRegistryActor ! PutMetadataAction(throwableToMetadataEvents(metadataKey, failure))
+  private def logCacheHitFailure(data: ResponsePendingData, reason: Throwable): ResponsePendingData = {
+    val problemSummary =
+      s"Failed copying cache results for job $jobDescriptorKey (${reason.getClass.getSimpleName}: ${reason.getMessage})"
+    if (invalidationRequired) {
+      // Whenever invalidating a cache result, always log why the invalidation occurred
+      workflowLogger.warn(        s"$problemSummary, invalidating cache entry.")
+      data
+    } else if (data.logCacheHitFailureCount < 3) {
+      workflowLogger.info(problemSummary)
+      data.copy(logCacheHitFailureCount = data.logCacheHitFailureCount + 1)
```

Comment:
TOL: Part of me likes the idea of:

* Always appending 1 to the count regardless 
* Renaming it something like `cacheHitFailureCount`
* Having a single log/metric emitted once the call cache sequence has completes, along the lines of `"Call cache hit process had ${data.cacheHitFailureCount} total hit failures before completing successfully/unsuccessfully"`